proto/src/main/proto/kvs.proto

```diff
@@ -436,6 +445,7 @@ message QueryPolicy {
   // Default: false
   optional bool failOnClusterChange = 11;
 
+  // Deprecated, use expectedDuration instead.
```

Comment:
@ashishshinde I added a "deprecated" comment, gRPC standard is to change the name of the field to include "deprecated" keyword (e.g. shortQueryDeprecated or something like that), wasn't sure if that's necessary for now.

Reply:
I guess this should be ok. Not sure what we plan to do when eventually.